dao/src/main/java/com/epam/esm/dao/sort/GiftCertificateSorter.java

```diff
@@ -0,0 +1,70 @@
+package com.epam.esm.dao.sort;
+
+import com.epam.esm.dao.entity.GiftCertificate;
+
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Order;
+import javax.persistence.criteria.Root;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GiftCertificateSorter {
+
+  public void sort(
+      CriteriaQuery<GiftCertificate> criteria,
```

Comment:
can't you combine root, builder, criteria? There are a lot of arguments in your methods

Reply:
For example you can get root from your criteria

Reply:
Yes, this is a good idea. (fixed)